### PR TITLE
Feat new version

### PR DESCRIPTION
**Pull request by isunaslabs**
Branch renamed from feat-new-code to feat-new-version
The overall code is still under development
NOTE: 
1.Typewriter effect on the hero component on every page is not added yet
2.Animation on the code sample view on the home page is not added yet

### CODE REVIEW

**Comment by JayJay1024** [comment]
LGTM

File: public/index.html [hunk 1]
```diff
@@ -40,7 +30,7 @@
     <title>Darwinia Network - 达尔文网络</title>
```
Comment (HackFisher) [inline public/index.html:30]:
```suggestion
    <title>Darwinia - Universal Cross-Chain Message Network</title>
```

File: public/index.html [hunk 2]
```diff
@@ -40,7 +30,7 @@
...
     <meta name="keywords" content="达尔文网络 darwinia 区块链 智能合约 token ethereum blockchain" />
```
Comment (HackFisher) [inline public/index.html:31]:
```suggestion
    <meta name="keywords" content="Darwinia Smart Contract Cross-chain Message Blockchain Substrate Polkadot" />
```

File: public/index.html [hunk 3]
```diff
@@ -40,7 +30,7 @@
...
     <meta name="description" content="Darwinia Network is a decentralized cross-chain bridge network building on Substrate, which is the &quot;Web 3.0 cross-chain bridge hub&quot; of the cross-chain ecology. It provides a safe and general bridging solution, connects to Polkadot, Ethereum, TRON, and other heterogeneous chains by cross-chain transfer of assets and remote chain calls. The application areas of Darwinia Network include Defi, cross-chain NFT trading, games, etc.">
```
Comment (HackFisher) [inline public/index.html:32]:
```suggestion
    <meta name="description" content="Darwinia Network Provides A Light Client-Based, Programmable, Universal Cross-Chain Message Network For DApp Developers.">
```

**Review by HackFisher — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 2 above)
Comment (isunaslabs) [reply public/index.html]: I just made a commit of your suggestions

File: public/index.html (hunk 3 above)
Comment (isunaslabs) [reply public/index.html]: I just made a commit of your suggestion

**Comment by fewensa** [comment]
It looks like you used hash-style route, you can delete it.
https://github.com/darwinia-network/home/blob/10c8896454388de2cda032da7ca16ecd4d6224ec/vercel.json#L26-L39
add it if you use history-style route (and update right uri path).
@isunaslabs

**Comment by fewensa** [comment]
Maybe this file/folder is not required? 🤔 
https://github.com/darwinia-network/home/blob/10c8896454388de2cda032da7ca16ecd4d6224ec/.vscode/settings.json#L1

**Comment by isunaslabs** [comment]
> Maybe this file/folder is not required? 🤔
> 
> https://github.com/darwinia-network/home/blob/10c8896454388de2cda032da7ca16ecd4d6224ec/.vscode/settings.json#L1

I understand. I'll delete that directory. Thanks for the feedback

**Comment by isunaslabs** [comment]
> 

Yeah! I noticed that the data in it maybe useless since I'm using the hash router but I left it just for the future reference.

**Review by 1022ranji — APPROVED** [review]

**Pull request merged**